docset/winserver2022-ps/failoverclusters/Test-Cluster.md

```diff
@@ -16,168 +16,90 @@ Runs validation tests for failover cluster hardware and settings.
 ## SYNTAX
 
 ```
-Test-Cluster [[-Node] <StringCollection>] [-Disk <Object[]>] [-Pool <Object[]>] [-ReportName <String>] [-List]
- [-Include <StringCollection>] [-Ignore <StringCollection>] [-Force] [-InputObject <PSObject>]
- [-Cluster <String>] [-WhatIf] [-Confirm] [<CommonParameters>]
+Test-Cluster [[-Node] <StringCollection>] [-Disk <Object[]>] [-Pool <Object[]>]
+[-ReportName <String>] [-List] [-Include <StringCollection>] [-Ignore <StringCollection>] [-Force]
+[-InputObject <PSObject>] [-Cluster <String>] [-WhatIf] [-Confirm] [<CommonParameters>]
```

Comment:
You need a leading space on each addtitional line in the syntax block for it to render properly.
```suggestion
Test-Cluster [[-Node] <StringCollection>] [-Disk <Object[]>] [-Pool <Object[]>]
 [-ReportName <String>] [-List] [-Include <StringCollection>] [-Ignore <StringCollection>] [-Force]
 [-InputObject <PSObject>] [-Cluster <String>] [-WhatIf] [-Confirm] [<CommonParameters>]
```